app_dart/lib/src/model/appengine/task.dart

```diff
@@ -178,6 +179,13 @@ class Task extends Model<int> {
   @JsonKey(name: 'Flaky')
   bool isFlaky;
 
+  /// Whether the test run of this task is flaky.
+  ///
+  /// Test runner supports rerun, and this flag tracks if a flake happens.
+  @BoolProperty(propertyName: 'TestRunFlaky')
+  @JsonKey(name: 'TestRunFlaky')
+  bool isTestRunFlaky;
```

Comment:
`isTestFlaky` is clearer to me (run was confusing me)

Reply:
Updated.